### DISCUSSION TITLE
IEA-15-240-RWT Data

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#2309] [General]
Hi all,
I’ve been using the IEA-15-240-RWT and the UMaine VolturnUS-S semisubmersible models to run some tests and simulations.
I was hoping to validate my work using the information from the “Definition of IEA-15-240-RWT” and the “Definition of the UMaine VolturnUS-S Reference Platform” documents. In particular, I was hoping to use the Power vs Wind speed curve provided in the document.
 
![image](https://github.com/OpenFAST/openfast/assets/71447919/93e984b3-10a0-4a0d-8ba0-7e7e92c555d5)

However I have had difficulties sourcing where this data has come from. Is it empirical? Is it sourced mathematically but uses empirical data in its calculations? Or is it purely mathematical?
Thank you!

**Comment by jjonkman** [comment]
Dear @Uisliu,

The figure you are showing from the IEA Wind 15-MW RWT specification report (https://www.nrel.gov/docs/fy20osti/75698.pdf) is numerical data of the steady-state solution as a function of wind speed generated by the original OpenFAST model of this wind turbine.

Best regards
